### PR TITLE
Added appId property and setCurrentAccessToken method

### PR DESCRIPTION
**Pull request by kopiro**
- Exposed the `appId` property (read by default from Info.split)
- Exposed the `setCurrentAccessToken` method to set an active access token for this session.

### CODE REVIEW

File: ios/Classes/FacebookModule.m [hunk 1]
```diff
@@ -195,6 +195,16 @@ -(id)accessToken
     }, YES);
     return token;
 }
+
+-(id)appID
+{
+    __block NSString * appID;
```
Comment (hansemannn) [inline ios/Classes/FacebookModule.m:185]: Does `ENSURE_UI_THREAD(...)` not work better here? No big fan of block states.

File: ios/Classes/FacebookModule.m [hunk 2]
```diff
@@ -740,6 +750,22 @@ -(void)requestWithGraphPath:(id)args
     }, NO);
 }
 
+-(void)setCurrentAccessToken:(id)args 
+{
+    ENSURE_SINGLE_ARG(args, NSString);
+
```
Comment (hansemannn) [inline ios/Classes/FacebookModule.m:740]: How can args be a String but contain Dictionary-values like below?

**Comment by hansemannn** [comment]
- CI failed
- Missing docs
- Missing example
- Version needs to be bumped
- Android implementation (if possible)

File: ios/Classes/FacebookModule.m (hunk 1 above)
Comment (kopiro) [inline ios/Classes/FacebookModule.m:185]: Ok! :)

File: ios/Classes/FacebookModule.m (hunk 2 above)
Comment (kopiro) [inline ios/Classes/FacebookModule.m:740]: Sorry,I forgot an older implementation.
Pushing again.

File: ios/Classes/FacebookModule.m (hunk 1 above)
Comment (kopiro) [inline ios/Classes/FacebookModule.m:185]: @hansemannn I copied the `accessToken` implementation.

```
-(id)accessToken
{
    __block NSString * token;
    TiThreadPerformOnMainThread(^{
        token = [[FBSDKAccessToken currentAccessToken] tokenString];
    }, YES);
    return token;
}
```

So, how should I change it?
Sorry, I'm not an ObjC developer :P

Comment (hansemannn) [inline ios/Classes/FacebookModule.m:185]: I just changed it here (from where you possible got it):

``` objc
-(id)permissions
{
    ENSURE_UI_THREAD_0_ARGS
    return [[[FBSDKAccessToken currentAccessToken] permissions] allObjects];
}
```

Does that work? Currently have no time to build and test. It just ensures it's on the right thread (main thread) and returns the value. Should be the same for the accessToken.

Comment (kopiro) [inline ios/Classes/FacebookModule.m:185]: Ok, I'll test it!

Comment (kopiro) [inline ios/Classes/FacebookModule.m:185]: It crash! :)

Comment (hansemannn) [inline ios/Classes/FacebookModule.m:185]: Ok, the leave it like it is for now

File: ios/Classes/FacebookModule.m [hunk 3]
```diff
@@ -162,14 +162,6 @@ -(id)canPresentShareDialog
...
 }
 
-/**
- * JS example:
- *
- * var facebook = require('facebook');
- * facebook.permissions = ['read_stream'];
- * alert(facebook.permissions);
- *
- */
```
Comment (hansemannn) [inline ios/Classes/FacebookModule.m:172]: Why did you remove the code comments here and below?

File: ios/Classes/FacebookModule.m [hunk 4]
```diff
@@ -179,22 +171,24 @@ -(id)permissions
...
 {
     __block NSString * token;
     TiThreadPerformOnMainThread(^{
-		token = [[FBSDKAccessToken currentAccessToken] tokenString];
+        token = [[FBSDKAccessToken currentAccessToken] tokenString];
     }, YES);
     return token;
 }
+
+-(id)appID
```
Comment (hansemannn) [inline ios/Classes/FacebookModule.m:183]: Missing code docs

**Comment by hansemannn** [comment]
Added some more comments. As soon as all comments are addressed, we can go ahead and merge for 5.4.0.

**Comment by hansemannn** [comment]
Removing PR from current milestone, since comments could not be addressed in time. Will revisit for a later release as soon as comments are addressed, no problem. Thanks!

**Comment by hansemannn** [comment]
Closing in favor of #80
